.github/workflows/tembo_release.yml

```diff
@@ -12,26 +12,70 @@ jobs:
 
   cargo_build:
     name: Cargo build
-    runs-on: ubuntu-20.04
-    container:
-      image: quay.io/tembo/muslrust:1.71.0-stable
     strategy:
       fail-fast: false
+      matrix:
+        include:
+          - target: x86_64-apple-darwin
+            asset_name: x86_64-apple
+            os: macos-latest
+          - target: aarch64-apple-darwin
+            asset_name: aarch64-apple
+            os: macos-latest
+          - target: x86_64-unknown-linux-musl
+            asset_name: x86_64-linux
+            os: ubuntu-20.04
+            container: quay.io/tembo/muslrust:1.71.0-stable
+          - target: aarch64-unknown-linux-musl
+            asset_name: aarch64-linux
+            os: ubuntu-20.04
+            container: messense/rust-musl-cross:aarch64-musl
+    runs-on: ${{ matrix.os }}
+    container:
+      image: ${{ matrix.container }}
     steps:
       - name: Check out the repo
         uses: actions/checkout@v4
 
+      - name: Install Deps Ubuntu
+        id: install_deps_ubuntu
+        run: |
+          set -x
+          apt update -y
+          apt install pkg-config libssl-dev -y
+        if: matrix.os == 'ubuntu-20.04'
```

Comment:
I suggest using `if: runner.os == 'Linux'` instead, so it should work without modification if you change the vision of Ubuntu.

Reply:
Yup, good idea. I will change that in the next PR.